### DISCUSSION TITLE
fastapi example wont work if nicegui should be the root page

### DISCUSSION THREAD

**Discussion by jkc20** [zauberzeug/nicegui#2959] [Q&A]
### Question

The fastapi example works out of the box, if nicegu is configured at /gui. As the comment suggested, one can omit the mount path "mount_path='/gui' " in frontend.py  to see the nicegui content at root. But localhost:8080 or localhost:8080/ gives still the swagger interface and it is not possible to access the nicegui part. Any suggestions how to solve it ?

**Comment by rodja** [comment]
What do you mean with "swagger"? Are you seeing the json `{'Hello': 'World'}`? There can only be one function to answer to a specific path. If you want to serve your UI on `/` the Json endpoints should have another path...

**Comment by jkc20** [comment]
HI Rodja,
thanks for your replay. It is working now.
It might be helpful to add a comment in main.py ?
@app.get('/') # has to be changed in case nicegui should be root
